var/spack/repos/builtin/packages/py-phylophlan/package.py

```diff
@@ -11,17 +12,25 @@ class PyPhylophlan(PythonPackage):
     phylogenetic profiling of genomes and metagenomes."""
 
     homepage = "https://github.com/biobakery/phylophlan"
-    url = "https://github.com/biobakery/phylophlan/archive/refs/tags/3.0.2.tar.gz"
+    url = "https://github.com/biobakery/phylophlan/archive/refs/tags/3.0.3.tar.gz"
 
+    version("3.0.3", sha256="d8d0082c95d58d7b11a60c1e2214b35c1a23a65675005f1393e7647d76c6a054")
     version("3.0.2", sha256="c342116662bbfbb49f0665291fc7c0be5a0d04a02a7be2da81de0322eb2256b4")
 
-    depends_on("python@3:", type=("build", "run"))
-    depends_on("py-numpy@1.12.1:", type=("build", "run"))
-    depends_on("py-biopython@1.70:", type=("build", "run"))
-    depends_on("py-dendropy@4.2.0:", type=("build", "run"))
-    depends_on("muscle", type=("build", "run"))
-    depends_on("blast-plus", type=("build", "run"))
-    depends_on("diamond", type=("build", "run"))
-    depends_on("py-matplotlib", type=("build", "run"))
-    depends_on("py-pandas", type=("build", "run"))
-    depends_on("py-seaborn", type=("build", "run"))
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-biopython@1.73:", type=("build", "run"))
+    depends_on("py-dendropy@4.4.0:", type=("build", "run"))
+    depends_on("py-matplotlib@3.1.0:", type=("build", "run"))
+    depends_on("py-numpy@1.15.4:", type=("build", "run"))
+    depends_on("py-pandas@0.24.2:", type=("build", "run"))
+    depends_on("py-seaborn@0.9.0:", type=("build", "run"))
+    depends_on("blast-plus@2.6.0:", type=("build", "run"))
+    depends_on("diamond@0.9:", type=("build", "run"))
+    depends_on("trimal@1.4.1:", type=("build", "run"))
+    depends_on("muscle@3.8.1551:", type=("build", "run"))
+    depends_on("mafft@7.310:", type=("build", "run"))
+    depends_on("fasttree@2.1.8:", type=("build", "run"))
+    depends_on("raxml@8.2.10:", type=("build", "run"))
```

Comment:
Where do all these dependency versions come from?

Reply:
https://github.com/bioconda/bioconda-recipes/blob/62ed7541de2a97cc0c9af553d6422fd143e912ce/recipes/phylophlan/meta.yaml

Reply:
The wiki just lists the tools (https://github.com/biobakery/phylophlan/wiki#external-tools) but for some reason the bioconda recipe is WAY more specific with versions.